INN/umbrella-citylimits
# Don't display newsletter popup box on homepage mobile

## Changes

This pull request makes the following changes:

- Updates the newsletter css to not display the newsletter popup on the mobile homepage

## Why



For #131

## Testing/Questions

Features that this PR affects:

- Homepage newsletter popup


Questions that need to be answered before merging:


Steps to test this PR:

1. Check out branch
2. Make sure the popup is not visible on the mobile homepage
